gcc-vm.yml

```diff
@@ -0,0 +1,147 @@
+      state: present
+
+- hosts: GCCVM
+  become: true
+  vars:
+          # created with:
+          # python -c 'import crypt; print crypt.crypt("1ns3cur3", "insecure")'
+    password: inyr/N2x97F.E
+
+  tasks:
+  - name: Install the latest version of git from wandisco repository
+    yum:
+      name: git
+      state: latest
+
+- hosts: GCCVM
+  vars:
+    ruby_version: "2.2.0"
+    rvm_path: "/home/rpm_omnibus/rvm/gems/ruby-{{ ruby_version }}/bin:/home/rpm_omnibus/rvm/gems/ruby-{{ ruby_version }}@global/bin:/home/rpm_omnibus/rvm/rubies/ruby-{{ ruby_version }}/bin:/home/rpm_omnibus/rvm/bin:/home/rpm_omnibus/gem/ruby/2.2.0/bin"
+    ruby_gem_path: "/home/rpm_omnibus/gem/ruby/2.2.0"
+
+  tasks:
+  - name: Append rvm path to environment
+    lineinfile: dest=/etc/environment state=present backrefs=yes regexp='PATH=(["]*)((?!.*?{{rvm_path}}).*?)(["]*)$' line="PATH=\1\2:{{rvm_path}}\3"
+
+  - name: Ensure necessary packages are installed
+    yum:
+      name: "{{ item }}"
+      state: present
+    with_items:
+      - curl
+      - gnupg2
+
+  - name: Ensure that GPG key for RVM is installed
+    shell: gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+    args:
+        creates: /root/.gnupg/secring.gpg
+    retries: 3
+    delay: 5
+    ignore_errors: yes
+
+  - name: Ensure that RVM is installed
+    shell: curl -L get.rvm.io | bash -s stable --path /home/rpm_omnibus/rvm
+    args:
+      creates: /home/rpm_omnibus/rvm
+
+  - name: Ensure we use RVM default paths
+    shell: source /home/rpm_omnibus/rvm/scripts/rvm
+
+  - name: Ensure that ruby is installed
+    command: "rvm install {{ ruby_version }}"
+    args:
+      creates: "/home/rpm_omnibus/rvm/gems/ruby-{{ ruby_version }}"
+    environment:
+      PATH: "{{ rvm_path }}:{{ ansible_env.PATH }}
+
+  - name: Set default version of ruby with rvm
+    command: "rvm alias create default ruby-{{ ruby_version }}"
+    args:
+      creates: /home/rpm_omnibus/rvm/config/alias
+    environment:
+      PATH: "{{ rvm_path }}:{{ ansible_env.PATH }}"
+
+  - name: Create bin directory in ruby directory
+    file:
+      path: "/home/rpm_omnibus/gem/ruby/{{ ruby_version }}/bin"
+      state: directory
+      owner: rpm_omnibus
+      group: rpm_omnibus
+      mode: 0775
+
+  - name: Create /opt/codethink-toolchain
+    file:
+      path: "/opt/codethink-toolchain"
+      state: directory
+      owner: rpm_omnibus
+      group: rpm_omnibus
+      mode: 0775
+    become: yes
+
+- hosts: GCCVM
+  vars:
+    ruby_version: "2.2.0"
+    ruby_path: "/home/rpm_omnibus/gem/ruby/2.2.0/bin"
+    ruby_gem_path: "/home/rpm_omnibus/gem/ruby/2.2.0"
+
+  tasks:
+  - name: Set the ruby version to use
+    command: rvm use "{{ ruby_version }}"
+
+  - name: Install bundler
+    shell: 'gem install --verbose {{ item }} --install-dir={{ ruby_gem_path }} --no-document --no-rdoc --no-ri --minimal-deps --platform ruby --no-wrappers --suggestions --clear-sources --backtrace'
```

Comment:
Could you use `http://docs.ansible.com/ansible/latest/gem_module.html` to install this?

Reply:
Gem module doesn't allow you to install the gem in a different path and in this case I would like to install the path where "install-dir" points to.